### DISCUSSION TITLE
Not able to reset password

### DISCUSSION THREAD

**Discussion by ghost** [community/community#84111] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi All,

I am not able to sign-in into my Github account. 
**When I try sign-up** it says "invalid or already taken". 
**When I try sign-in/reset password** it says "invalid or not associated with personal account or organization email only for notifications"

<img width="281" alt="Screenshot 2024-01-04 at 3 09 49 PM" src="https://github.com/community/community/assets/78838887/aff1ae8b-ec7a-4064-9eba-dc4d58d9a0ad">

<img width="1214" alt="Screenshot 2024-01-04 at 3 09 27 PM" src="https://github.com/community/community/assets/78838887/87380ab4-3fd6-4ed6-af53-008518b4339b">

Please let me know what is the issue here. Thanks

**Comment by nandparticles** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> Hi All,
> 
> I am not able to sign-in into my Github account. 
> **When I try sign-up** it says "invalid or already taken". 
> **When I try sign-in/reset password** it says "invalid or not associated with personal account or organization email only for notifications"
> 
> 
> <img width="281" alt="Screenshot 2024-01-04 at 3 09 49 PM" src="https://github.com/community/community/assets/78838887/aff1ae8b-ec7a-4064-9eba-dc4d58d9a0ad">
> 
> <img width="1214" alt="Screenshot 2024-01-04 at 3 09 27 PM" src="https://github.com/community/community/assets/78838887/87380ab4-3fd6-4ed6-af53-008518b4339b">
> 
> 
> Please let me know what is the issue here. Thanks
> 

Same thing is happening to me and i already lost 2 accounts because of this
